packages/next-sample/pages/index.tsx

```diff
@@ -5,16 +5,17 @@ import useSWR from 'swr';
 
 const Home = () => {
   const { data } = useSWR<LogtoContext>('/api/logto/user');
+  const { data: dataWithUserInfo } = useSWR<LogtoContext>('/api/logto/user-info');
```

Comment:
Seems we also need to update these two routes

Reply:
`/api/logto/user` is the SDK's built-in route (also recommended), so I don't change the name, instead, I add 2 comments on them.